js/pokedex-pokemon.js

```diff
@@ -254,7 +254,7 @@ var PokedexPokemonPanel = PokedexResultPanel.extend({
 			if (typeof sources === 'string') sources = [sources];
 			for (var i=0, len=sources.length; i<len; i++) {
 				var source = sources[i];
-				if (source.substr(0,2) === '7L') {
+				if (source.substr(0,2) === '8L') {
```

Comment:
Should we show gen-7 learnsets for old mons?

Reply:
did